backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubUploaderService.java

```diff
@@ -114,26 +205,92 @@ public String creatTestCVSForDataHub(String lastEndCreateOn) {
     // There is also the risk of the top action running multiple times concurrently.
     // ultimately, it would be nice if each row had an ID that could be dedupped on the server.
     @Transactional(readOnly = true)
-    public Map<String,String> uploadTestEventCVSToDataHub(final String apiKey, String lastEndCreateOn) {
+    public Map<String, String> uploadTestEventCSVToDataHub(final String apiKey, String lastEndCreateOn) {
         try {
-            this._createTestEventCSV(lastEndCreateOn);
-            this._uploadCSVDocument(apiKey);
+            this.createTestEventCSV(lastEndCreateOn);
+            this.uploadCSVDocument(apiKey);
 
-            return  Map.of(
+            return Map.of(
```

Comment:
If we know the fields at this point, we can/probably should turn this into a plain Java object instead of a map.

Reply:
Seems like this could be tagged as part of #457 as well.